exonum-java-binding/core/src/main/java/com/exonum/binding/core/transaction/InternalTransactionContext.java

```diff
@@ -21,6 +21,7 @@
 import com.exonum.binding.common.crypto.PublicKey;
 import com.exonum.binding.common.hash.HashCode;
 import com.exonum.binding.core.storage.database.Fork;
+import java.util.Objects;
 
 /**
  * Default implementation of the transaction context.
```

Comment:
Would it make sense to use `@AutoValue` here?

Reply:
Sure, added `@AutoValue`.